pkg/adapter/config/kubernetes/sensitiveDoguConfigRepository.go

```diff
@@ -0,0 +1,52 @@
+package kubernetes
```

Comment:
🚫 **[checkstyle]** <dupl> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>1-52 lines are duplicate of `pkg/adapter/config/kubernetes/doguConfigRepository.go:1-52`
<!-- __reviewdog__:ChA2ZDM4ZjZmYjM2ODZmNDc4EgpjaGVja3N0eWxl -->